examples/timestamping/frontend/src/plugins/blockchain.js

```diff
@@ -3,88 +3,77 @@ import axios from 'axios'
 import * as proto from '../../proto/stubs.js'
 
 const PER_PAGE = 10
-const SERVICE_ID = 130
+const SERVICE_ID = 4
 const TX_ID = 0
-const TABLE_INDEX = 0
 const TimestampEntry = Exonum.newType(proto.exonum.examples.timestamping.TimestampEntry)
 
 module.exports = {
-  install(Vue) {
+  install (Vue) {
     Vue.prototype.$blockchain = {
-      generateKeyPair() {
+      generateKeyPair () {
         return Exonum.keyPair()
       },
 
       createTimestamp: (keyPair, hash, metadata) => {
         // Describe transaction
-        const transaction = Exonum.newTransaction({
-          author: keyPair.publicKey,
-          service_id: SERVICE_ID,
-          message_id: TX_ID,
-          schema: proto.exonum.examples.timestamping.TxTimestamp
+        const transaction = new Exonum.Transaction({
```

Comment:
Don't need to reuse transaction in this case, so no reasons to move it into an uppermost scope.

Reply:
Well, it will be initialized once when the module is loaded instead of every time the method with the current definition is called (unless the JS runtime is smart enough to move it upward itself) 🙃